src/mlpack/tests/split_data_test.cpp

```diff
@@ -212,3 +212,114 @@ TEST_CASE("SplitLabeledDataLargerTest", "[SplitDataTest]")
 
   CheckDuplication(std::get<2>(value), std::get<3>(value));
 }
+
+/**
+ * Check that 0 test ratio results in a full train set for stratified split
```

Comment:
```suggestion
 * Check that 0 test ratio results in a full train set for stratified split.
```

Missing stop at the end.